requirements/requirements.txt

```diff
@@ -3,5 +3,5 @@ pexpect~=4.8
 requests~=2.26
 json_database~=0.7
 kthread~=0.2
-ovos_config~=0.0,>=0.0.3
+ovos_config~=0.0,>=0.0.6a1
```

Comment:
It would still be a manual thing, but we could partially resolve the issue of pinned prereleases by pinning an unreleased max version. At least the compatible version, less than the next potentially incompatible bump?

Reply:
That would make it difficult for dev workflows since a change to ovos-config would then need to generate a dependency bump in utils (or worse, a bump to utils needing changes to EVERY ovos package for dependency resolution).